src/mtype.h

```diff
@@ -144,6 +144,7 @@ enum m_flag : int {
     MF_INTERIOR_AMMO,       // Monster contain's its ammo inside itself, no need to load on launch. Prevents ammo from being dropped on disable.
     MF_CLIMBS,              // Monsters that can climb certain terrain and furniture
     MF_PACIFIST,            // Monsters that will never use melee attack, useful for having them use grab without attacking the player
+    MF_KEEP_DISTANCE,       // Attempts to keep a short distance from its current target.
```

Comment:
```suggestion
    MF_KEEP_DISTANCE,       // Attempts to keep a short distance (tracking_distance) from its current target.
```
Maybe? And/or mention 8 as default?